point-of-sale-app/frontend/src/main/java/com/google/abmedge/frontend/FrontendController.java

```diff
@@ -45,21 +59,23 @@
 @RestController
 public class FrontendController {
 
-  private static final Logger LOGGER =
-      LogManager.getLogger(FrontendController.class);
+  private static final Logger LOGGER = LogManager.getLogger(FrontendController.class);
+  private static final String FAILED = "Failed to fetch items; try again after a while";
   private static final String INVENTORY_EP_ENV = "INVENTORY_EP";
   private static final String PAYMENTS_EP_ENV = "PAYMENTS_EP";
-  private static final String FAILED = "Failed to fetch items; try again after a while";
   private static final String ITEMS_EP = "/items";
+  private static final String ITEMS_BY_ID_EP = "/items_by_id";
   private static final String SWITCH_EP = "/switch";
   private static final String UPDATE_EP = "/update";
   private static final String PAY_EP = "/pay";
-  private static String INVENTORY_SERVICE = "http://inventory-svc:8080";
-  private static String PAYMENTS_SERVICE = "http://payments-svc:8080";
   private static final HttpClient HTTP_CLIENT = HttpClient.newBuilder()
       .version(HttpClient.Version.HTTP_1_1)
       .connectTimeout(Duration.ofSeconds(10))
       .build();
+  private static String INVENTORY_SERVICE = "http://inventory-svc:8080";
+  private static String PAYMENTS_SERVICE = "http://payments-svc:8080";
```

Comment:
Would it be possible to use dynamic env variables instead of hardcoding the service name and port?

Reply:
These are only default values. On startup these endpoints are read from environment variables via the [`initServiceEndpoints` method](https://github.com/GoogleCloudPlatform/anthos-edge-usecases/blob/frontend-pay-api/point-of-sale-app/frontend/src/main/java/com/google/abmedge/frontend/FrontendController.java#L217-L234). However, I have added a comment explaining this in https://github.com/GoogleCloudPlatform/anthos-edge-usecases/pull/5/commits/64a08c0dcf6bdf5694d6d885bb15b4bb6ab40a2a